modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/java/com/liferay/dynamic/data/mapping/form/builder/internal/servlet/DDMFormFunctionsServlet.java

```diff
@@ -64,58 +64,53 @@ protected void doGet(
 			HttpServletResponse httpServletResponse)
 		throws IOException, ServletException {
 
-		Map<String, DDMExpressionFunction> ddmExpressionFunctions = null;
+		Map<String, DDMExpressionFunctionFactory> ddmExpressionFunctions = null;
```

Comment:
Change the name of the variable since you're not returning `DDMExpressionsFunction` anymore. See the comments below.